allel/stats/diversity.py

```diff
@@ -224,7 +225,7 @@ def sequence_diversity(pos, ac, start=None, stop=None,
     Returns
     -------
 
-    pi : ndarray, float, shape (n_windows,)
+    pi : float
```

Comment:
why this change?

Reply:
because sequence_diversity and divergence return a single float, not in windows

Reply:
why, so it does. =)

Reply:
oh, and that's #273, huh.